packages/vue/src/router.ts

```diff
@@ -92,7 +92,9 @@ export function vueRouterInstrumentation(router: VueRouter): VueRouterInstrument
       if (startTransactionOnPageLoad && isPageLoadNavigation) {
         const pageloadTransaction = getActiveTransaction();
         if (pageloadTransaction) {
-          pageloadTransaction.setName(transactionName, transactionSource);
+          if (pageloadTransaction.metadata.source !== 'custom') {
+            pageloadTransaction.setName(transactionName, transactionSource);
+          }
           pageloadTransaction.setData('params', data.params);
           pageloadTransaction.setData('query', data.query);
```

Comment:
I guess there's an argument to be made that if we have a custom source here, we don't need to set the `params` and `query` data. Happy to change this if reviewers prefer that.

Reply:
I'm generally of the school of thought that (within reason) more data is better. We've been sending it before now, so I vote that we keep doing so.

Reply:
Good point. Then I'll keep it as is